test/unit/android-controller-specs.js

```diff
@@ -3,11 +3,16 @@
 var chai = require('chai')
   , controller_path = '../../lib/devices/android/android-controller.js'
   , controller = require(controller_path)
-  , _ = require('underscore');
+  , _ = require('underscore')
+  , sinon = require('sinon');
 
 chai.should();
 
 describe('android-controller', function () {
+  beforeEach(function() {
+    controller.adb = { getApiLevel: function() {} };
```

Comment:
Or to replace the stubbing below to use this direct definition instead of sinon would work as well.  This would limit the scope of the mocking.